taca/utils/bioinfo_tab.py

```diff
@@ -116,17 +130,18 @@ def get_status(run_dir):
 
     return status
 
-
-
-
+#Returns project, FC, lane & sample (sample-run) status for a given folder
 def get_ss_projects(run_dir):
-    pattern=re.compile("(P[0-9]{3,5})_[0-9]{3,5}")
-    project_ids=set()
+    proj_tree = Tree()
+    proj_pattern=re.compile("(P[0-9]{3,5})_[0-9]{3,5}")
```

Comment:
The regex matches a sample but the stored var is the project. 
As it was written like this before me (probably to avoid some conflicting hits) I see no reason to change it.